nano/core_test/conflicts.cpp

```diff
@@ -54,92 +54,87 @@ TEST (conflicts, add_two)
 	nano::system system{};
 	auto const & node = system.add_node ();
 
+	system.wallet (0)->insert_adhoc (nano::dev::genesis_key.prv);
```

Comment:
This is a big test. An overview describing what is happening and why would be useful.

Reply:
Let's tackle overviews with everyone else.

I'll be pushing a few more test fixes but unless the description of the test is trivial, without such description for now.